### PR TITLE
remove the hardcoded limit for highlights

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/2561

Rather than hardcode the limit, it's nicer to have whoever updates the initiatives page decide how much content should be shown.

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html [hunk 1]
```diff
@@ -72,11 +72,9 @@ <h3 class="h2-heading">{{ page.h3 }}</h3>
   <div class="highlights mb-5">
     <div class="row">
       {% for highlight in page.featured_highlights.all %}
-      {% if forloop.counter <= 3 %}
       <div class="col-md-6 col-lg-4 {% if forloop.counter == 3 %}d-md-none d-lg-block{% endif %}">
```
Comment (alanmoo) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html]: It's not obvious to me what this is doing, but...I feel like it needs to be changed too. Yes/no?

Comment (alanmoo) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html]: (The forloop.counter check)

Comment (Pomax) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html]: it just goes "is this highlight 0, 1, or 2? Cool, make a div" and even if there are more, it won't build divs for them.

Comment (alanmoo) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html]: False: See medium viewports (this is a bootstrap class, the `-md-` tipped me off). Following up in a comment, as we need to pull in Kristina.

Comment (Pomax) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/static/initiatives_page.html]: gah, always forget about those breakpoints =_=

looks like the removal got okayed, though, so I shall land this.

**Pull request merged**